packages/paste-website/src/pages/components/disclosure/index.mdx

```diff
@@ -241,6 +242,36 @@ You can control the initial state of the disclosure to be expanded or collapsed
 </Disclosure>`}
 </LivePreview>
 
+### Using state hooks
+
+<Callout variant="warning">
+  <CalloutTitle>🚨 ATTENTION!! 🚨</CalloutTitle>
+  <CalloutText>
+    If you decided to use state hooks, the Paste team is <strong>NOT</strong> responsible for debugging any issues in
+    your application.
+  </CalloutText>
+</Callout>
+
+The `state` prop allows you to hook into, and keep in sync the state of the Disclosure and the state of your application.
+
+This will allow you to set initial state and merge the returned state from the hook with the application state,
+and keep it in sync with user interactions.
+
+In the example below, we're using a `state` to control visibility, and setting a transition delay of one second.
```

Comment:
This isn't necessarily true, I mean it kind of is as we delay the showing of the disclosure but we're not transitioning anything. We're also not doing anything with the transition state that we're creating. 

For this and the story, as a reader I'm not seeing the direct effect of setting extra state. In Reakit they at least use it to show a loading message before the toggle happens so I can see the connection directly in the example.

I'd suggest reworking this example in the docs and stories to be a little more visible of the effect you can take with mixed in state.

Reply:
I've updated the story and doc example. I also changed the example copy to use a more real world idea. Let me know if I should add more to that.